nupic/frameworks/opf/clamodel.py

```diff
@@ -419,7 +423,7 @@ def run(self, inputRecord):
     if self.__logger.isEnabledFor(logging.DEBUG):
       self.__logger.debug("inputRecord: %r, results: %r" % (inputRecord,
                                                             results))
-
+    results.iterationTime = time.clock() - startTime
```

Comment:
yes, I want to create a new metric that has access to this information, since metric is called from swarming, the process of calling `model.run()` has to be same, I cannot use an alternative path..or?

> this metric would expose time per iteration from Model.run(), I imagine this combined with #1712 like `CombinedMetric(TimeMetric(window=50), 0.3 (=weight), altMAPE(window=50), 0.7)`